rotkehlchen/chain/evm/decoding/stakedao/utils.py

```diff
@@ -20,24 +26,35 @@
 SUPPORTED_STAKEDAO_STRATEGIES: Final = {'curve', 'balancer', 'pendle', 'pancakeswap', 'angle'}
 
 
-def query_stakedao_gauges() -> None:
+def query_stakedao_gauges(evm_inquirer: 'EvmNodeInquirer') -> None:
     """Query StakeDAO gauges for lockers and strategies from the API and cache them."""
-    all_gauges: dict[ChainID, set] = defaultdict(set)
-
+    all_gauges: set[tuple[ChecksumEvmAddress, ChecksumEvmAddress]] = set()
     try:  # Query lockers
```

Comment:
this try/except checks for keyError but you also catch keyError in line 43. Maybe you don't want to have the general try/except or the inner try/except